proto/meta.proto

```diff
@@ -156,10 +156,19 @@ message ListTableFragmentsResponse {
   map<uint32, TableFragmentInfo> table_fragments = 1;
 }
 
+message ListTablesRequest {
+  repeated uint32 table_ids = 1;
+}
+
+message ListTablesResponse {
+  map<uint32, catalog.Table> tables = 1;
+}
+
 service StreamManagerService {
   rpc Flush(FlushRequest) returns (FlushResponse);
   rpc CancelCreatingJobs(CancelCreatingJobsRequest) returns (CancelCreatingJobsResponse);
   rpc ListTableFragments(ListTableFragmentsRequest) returns (ListTableFragmentsResponse);
+  rpc ListTables(ListTablesRequest) returns (ListTablesResponse);
```

Comment:
minor: I think it is more appropriate to put the RPC in DdlService (in ddl_service.proto) since all catalog related stuff are in there.

Reply:
done